File: server/model/hotel.js
const Sequelize = require('sequelize')
const MySequelize = require('../config/mysqlConfig')

const hotelModel = MySequelize.define('hotel', {
  id: {
    type: Sequelize.STRING(19),
    field:  'id',
    primaryKey: true
  },
  name: {
    type: Sequelize.STRING(20),
    field:  'name',
    allowNull: false
  },
  description: {
    type: Sequelize.STRING(500),
    field:  'description',
    allowNull: false
  },
  cover: {
    type: Sequelize.STRING(255),
    field:  'cover',
    allowNull: false
  },
  recommend_rank: {
    type: Sequelize.TINYINT,
    field:  'recommend_rank',
    allowNull: false
  },
  address: {
    type: Sequelize.STRING(255),
    field:  'address',
    allowNull: false
  },
  telephone: {
    type: Sequelize.STRING(11),
    field:  'telephone',
    allowNull: false
  },
  is_deleted: {
    type: Sequelize.TINYINT,
    field:  'is_deleted',
    allowNull: false
  },
  create_time: {
    type: Sequelize.DATE,
    field:  'create_time',
    allowNull: false
  },
  modify_time: {
    type: Sequelize.DATE,
    field:  'modify_time',
    allowNull: false
  },
})

module.exports = hotelModel

File: src/main/java/com/carolin_violet/travel_system/bean/Hotel.java
package com.carolin_violet.travel_system.bean;

import com.baomidou.mybatisplus.annotation.*;

import java.util.Date;
import java.io.Serializable;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

/**
 * <p>
 * 
 * </p>
 *
 * @author carolin-violet
 * @since 2022-05-22
 */
@Data
@EqualsAndHashCode(callSuper = false)
@Accessors(chain = true)
public class Hotel implements Serializable {

    private static final long serialVersionUID = 1L;

    /**
     * 旅馆id
     */
    @TableId(value = "id", type = IdType.ID_WORKER_STR)
    private String id;

    /**
     * 旅馆名称
     */
    private String name;

    /**
     * 旅馆介绍
     */
    private String description;

    /**
     * 旅馆封面展示图片
     */
    private String cover;

    /**
     * 推荐指数(1,2,3,4,5)
     */
    private Integer recommendRank;

    private String telephone;

    private String address;

    /**
     * 逻辑删除
     */
    @TableLogic
    @TableField(fill = FieldFill.INSERT)
    private Integer isDeleted;

    /**
     * 添加时间
     */
    @TableField(fill = FieldFill.INSERT)
    private Date createTime;

    /**
     * 修改时间
     */
    @TableField(fill = FieldFill.INSERT_UPDATE)
    private Date modifyTime;


}
